frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -99,24 +96,27 @@
               <hr class="mt-6" />
               <h4 id="how-it-was-made" class="is-info is-size-4">How it was made</h4>
               <p>
-                Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a
-                piece of classical Latin literature from 45 BC, making it over 2000 years old.
-                Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked
-                up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and
-                going through the cites of the word in classical literature, discovered the
-                undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de
-                Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45
-                BC. This book is a treatise on the theory of ethics, very popular during the
-                Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes
-                from a line in section 1.10.32.
+                Data in this database are predicted from available advanced deep learning tools:
+                k<sub>cat</sub> published in <i>Nature Catalysis</i> in 2022 (<a
+                  href="https://doi.org/10.1038/s41929-022-00798-z"
+                  >doi:10.1038/s41929-022-00798-z</a
+                >). Our vision for this database is to store every possible predicted parameter for
+                any available enzyme to facilitate
```

Comment:
Should this sentence really end here..?

Reply:
Great that you caught this @MalinAhlberg. I think the original was `to facilitate better science`, but I think it's time to address @inghylt's previous comment [here](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1031#discussion_r898955659).

```suggestion
                any available enzyme to facilitate improved predictions.
```

What do you think?

Reply:
The new version sounds really nice! The old one is also fine to me.

Reply:
It is not improved predictions. We can skip the part 'to facilitate...', just `our vision is to store every possible predicted parameter for any available enzyme`